src/entities/uniqueAssets.ts

```diff
@@ -62,3 +58,9 @@ export interface UniqueAsset {
   isPoap?: boolean;
   network?: Network;
 }
+
+export interface UniqueAssetTrait {
+  trait_type: string;
+  value: string | number;
+  display_type: string;
```

Comment:
...`text`?

Reply:
how about `title`? I think this should be in trait. The doc is not exhaustive in types I guess?

Reply:
It is not in OpenSea docs, but I fuess we could add it as an optional if it gets returned

Reply:
I don't think anything else belongs here to be honest. We already have everything required, some of the trait values are renamed later mapped to something else in a tag

Reply:
I added `max_value` since given commit history it was used byt it is not documented by opensea api docs. Since it is optional it shouldn't cause any problem being there.